codegen/templates/endpoint_register.tmpl

```diff
@@ -34,7 +34,7 @@ func CreateEndpoints(
 // Register will register all endpoints
 func Register(g *zanzibar.Gateway) {
 	router := g.Router
-	tchannelServer := g.TChannelServer
+	tchanRouter := g.TChanRouter
```

Comment:
The abbreviation `TChan` seems excessive in field names, we can use `TChannelRouter`.